docs/source/gs-processing/developer/input-configuration.rst

```diff
@@ -373,6 +372,35 @@ arguments.
          split the values in the column and create a vector column
          output. Example: for a separator ``'|'`` the CSV value
          ``1|2|3`` would be transformed to a vector, ``[1, 2, 3]``.
+-  ``numerical``
+
+   -  Transforms a numerical column using a missing data imputer and an
+      optional normalizer.
+   -  ``kwargs``:
+
+      -  ``imputer`` (String, optional): A method to fill in missing values in the data.
+         Valid values are:
+         ``none``, ``mean`` (Default), ``median``, and ``most_frequent``. Missing values will be replaced
+         with the respective value computed from the data.
+      - ``normalizer`` (String, optional): Applies a normalization to the data, after
+         imputation. Can take the following values:
+         - ``none``: (Default) Don't normalize the numerical values during encoding.
+         - ``min-max``: Normalize each value by subtracting the minimum value from it,
+        and then dividing it by the difference between the maximum value and the minimum.
+        - ``standard``: Normalize each value by dividing it by the sum of all the values.
+-  ``multi-numerical``
+
+   -  Column-wise transformation for vector-like numerical data using a missing data imputer and an
```

Comment:
We have the option to define it as `none` would you like us to make the default? I'm following NeptuneML here, and it's the "safest" choice for users (otherwise the user will figure out the issue way down the pipeline).

Reply:
According to https://github.com/awslabs/graphstorm/pull/575/files#diff-ec035bfb83573a8c31659fd21e4d5211df702c12f6199ee764e3bfab3d2ca846R46 We are now use none as the default value, can you also update the doc accordingly?